cmd/security-agent/app/app.go

```diff
@@ -143,16 +145,6 @@ func start(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	// Setup healthcheck port
-	var healthPort = coreconfig.Datadog.GetInt("health_port")
```

Comment:
I think we should still have a heath_port for every agent.
But a dedicated port for each agent.

maybe `security-agent.health_port`
wdyt?

Reply:
Currently none of our agents (except core) has it, so I'd rather have the security-agent aligned. We can implement a proper solution for all agents in an upcoming PR.